### ISSUE TITLE
Question: Functional API Validation Data

### ISSUE THREAD

**Issue by ADulian** [keras-team/keras#12453]
Hi there,

Quick question regarding use of functional API.

At the moment I'm working with multi-input model consisting of 2 heads (CNN and RNN). I'm not entirely sure how to work around the feeding of validation data into the model, i.e. in terms of using a single train set without validation set the fit function looks like this:

model.fit([X_one, X_two], Y, epochs...) 

what I want is to achieve something like the example below:
model.fit([X_one_train, X_two_train], [Y_one_train, Y_two_train], validation_data=([X_one_val, X_two_val],[Y_one_val, Y_two_val]))

but unfortunately so far I have not been able to find a correct way of training the model that way  

Appreciate any help,
Regards,

**Comment by goodyduru** [comment]
You could check [this](https://keras.io/getting-started/functional-api-guide/) out under the section titled Multi-input and multi-output models
